sdk/src/test/java/io/dapr/client/DaprClientHttpTest.java

```diff
@@ -388,6 +390,28 @@ public void invokeServiceNoHotMono() {
     // No exception should be thrown because did not call block() on mono above.
   }
 
+  @Test
+  public void invokeServiceWithContext() {
```

Comment:
Does this test actually check that the header is filtered out? I'm not sure if the `mockInterceptor` rule will only be matched if it has just those headers.